easybuild/easyconfigs/b/BuildEnv/BuildEnv-1.0-intel-2015a-default.eb

```diff
@@ -0,0 +1,13 @@
+easyblock = "BuildEnv"
+
+name = 'BuildEnv'
+version = '1.0'
+versionsuffix = '-default'
```

Comment:
The idea was that this file is for the default toolchain options, but you could set different toolchain options which would lead to different suffixes.